.github/workflows/generate-manual.yaml

```
@@ -0,0 +1,76 @@
+
+      - name: Create Pull Request
+        if: steps.check_changes.outputs.changes == 'true'
+        uses: peter-evans/create-pull-request@v5
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          branch: update-generated-files
+          base: ${{ github.event.inputs.branch }}
+          title: "ci: generate files"
+          body: |
+            This PR updates the auto-generated files with `yarn generate`. Please review the changes.
+          labels: ["CI"]
```

actionlint

expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)